src/repositories/meteorsRepository.js

```diff
@@ -1,7 +1,10 @@
+const { NASA_API_URL } = require('../configurations/config');
 const axios = require('axios');
+const NASA_API_NEO_URL = NASA_API_URL + '/neo/rest/v1/feed';
+
 
 const getMeteorsData = async (startDate, endDate) => {
-    const url = `${process.env.NASA_API_NEO_URL}?start_date=${startDate}&end_date=${endDate}&api_key=${process.env.NASA_API_KEY}`;
+    const url = `${NASA_API_NEO_URL}?start_date=${startDate}&end_date=${endDate}&api_key=${process.env.NASA_API_KEY}`;
```

Comment:
add please config file for environment variables manipulations

Reply:
@Mikhail1992 I have config.js file already, I just forgot to import process.env.NASA_API_KEY. 
Fixed